logger_core/Cargo.toml

```diff
@@ -13,7 +13,7 @@ test-env-helpers = "0.2.2"
 
 [dependencies]
 tracing = "0.1"
-tracing-appender = "0.2.2"
+tracing-appender = { version = "0.2.3", default-features = false }
```

Comment:
Did you check if it has any implication to the behavior, does the features connected to our usage in some way?
Generally for faster builds we should disable default feature everywhere.
We'll not go all over the packages now (for redis-rs ill do it at some point) but let's try to make sure we do that when adding new packages.

Reply:
there's no implications
https://github.com/tokio-rs/tracing/blob/master/tracing-appender/Cargo.toml